app/src/main/scala/org/alephium/explorer/persistence/schema/BlockDepsSchema.scala

```diff
@@ -21,14 +21,14 @@ import slick.jdbc.JdbcProfile
 import slick.lifted.{Index, PrimaryKey, ProvenShape}
 
 import org.alephium.explorer.api.model.BlockEntry
+import org.alephium.explorer.persistence.model.BlockDepEntity
 
 trait BlockDepsSchema extends CustomTypes {
   val config: DatabaseConfig[JdbcProfile]
 
   import config.profile.api._
 
-  class BlockDeps(tag: Tag)
-      extends Table[(BlockEntry.Hash, BlockEntry.Hash, Int)](tag, "block_deps") {
+  class BlockDeps(tag: Tag) extends Table[BlockDepEntity](tag, "block_deps") {
     def hash: Rep[BlockEntry.Hash] = column[BlockEntry.Hash]("hash", O.SqlType("BYTEA"))
     def dep: Rep[BlockEntry.Hash]  = column[BlockEntry.Hash]("dep", O.SqlType("BYTEA"))
     def order: Rep[Int]            = column[Int]("order")
```

Comment:
Shall we name `order` as `dep_order`? I remember you had issues using `order`

Reply:
yes good point, `order` is sql keyword.